### PR TITLE
Semantic release

### PR DESCRIPTION
**Pull request by rolandszoke**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 version: 2
 jobs:
-  build:
+  test:
     docker:
       # specify the version you desire here
       - image: circleci/node:8.12.0
```
Comment (solkimicreb) [inline .circleci/config.yml]: Please bump the node version to 10 here too (to be consistent with the release Node version).

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@risingstack/react-easy-state",
-  "version": "6.1.3",
+  "version": "0.0.0-development",
```
Comment (solkimicreb) [inline package.json]: Why did you rewrite this to a new version? We had a vote about this on Slack and we decided to continue versioning from the current version (6.1.3). This version is already published to npm with an alpha tag.

**Comment by solkimicreb** [comment]
I think we should not merge this until https://github.com/RisingStack/react-easy-state/pull/127 is merged. (We don't want to send release new versions of the old package, instead we want people to migrate to the @risingstack package).

File: package.json (hunk 2 above)
Comment (rolandszoke) [reply package.json]: This is a semantic-release configuration requirement. From now on sematic-release will be responsible for the versioning, and this version field will be constant.

**Pull request merged**